src/shared/AlertWithConfirmation/index.jsx

```diff
@@ -0,0 +1,49 @@
+// eslint-disable-next-line no-unused-vars
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
+
+//this is taken from https://designsystem.digital.gov/components/alerts/
+class AlertWithConfirmation extends Component {
+  cancelActionHandler = () => {
+    return this.props.cancelActionHandler;
+  };
+
+  confirmActionHandler = () => {
+    return this.props.okActionHandler;
+  };
+
+  render() {
+    return (
+      <div className={`usa-alert usa-alert-${this.props.type} usa-width-one-whole`}>
+        <div className="usa-alert-body usa-width-one-whole">
+          <div className="body--heading">
+            <div>{this.props.heading && <h3 className="usa-alert-heading">{this.props.heading}</h3>}</div>
+            <div className="usa-alert-text">{this.props.message}</div>
+            <div className="cancel-or-ok-buttons">
+              <button type="button" className="usa-button-secondary" onClick={this.cancelActionHandler()}>
+                Cancel
+              </button>
+              <button type="button" className="usa-button" onClick={this.confirmActionHandler()}>
```

Comment:
We can just use the props we passed in to the component for the onClick handler 

```suggestion
              <button type="button" className="usa-button-secondary" onClick={this.props.okActionHandler}>
```